source/kraken/fill_disruption_from_chaos.cpp

```diff
@@ -443,79 +691,118 @@ struct add_impacts_visitor : public apply_impacts_visitor {
         return true;
     }
 
-    void operator()(const nt::StopPoint* stop_point) {
-        this->log_start_action(stop_point->uri);
-        if (stop_point->journey_pattern_point_list.empty()) {
+    void remove_jp_if_no_vj(nt::JourneyPattern* jp, const std::set<nt::new_disruption::Impact*>& jp_impacts) {
```

Comment:
The pure "remove jp" part has to be in PT_Data (which has impacts, thus this parameter won't be needed)